content/handbook/product-development/how-we-work/product-development-flow/feature-flag-lifecycle/_index.md

```
@@ -0,0 +1,224 @@
+
+## 機能フラグの使い方
+
+GitLab 開発で機能フラグを始める概要については、[トレーニングテンプレート](https://gitlab.com/gitlab-com/www-gitlab-com/-/blob/master/.gitlab/issue_templates/feature-flag-training.md) に従ってください。
+
+機能フラグを使用する前に、次のガイダンスを必ず読んでください：
+
+1. [機能フラグを使った開発](https://docs.gitlab.com/ee/development/feature_flags/)：機能フラグのタイプ、その定義と検証、作成方法、フロントエンドとバックエンドの詳細、その他の情報について学びます。
+1. [機能フラグの背後でデプロイされた機能のドキュメント化](https://docs.gitlab.com/ee/development/documentation/feature_flags.html)：状態に応じて機能フラグの背後にデプロイされた機能をドキュメント化する方法と、状態が変わったときにドキュメントを更新する方法。
+1. [機能フラグの制御](https://docs.gitlab.com/ee/development/feature_flags/controls.html)：新機能のデプロイ、GitLab.com での有効化、変更のコミュニケーション、ロギング、クリーンアップのプロセスについて学びます。
+1. [機能フラグと変更管理プロセス](/handbook/engineering/infrastructure-platforms/change-management/#feature-flags-and-the-change-management-process)：あなたのフラグが変更管理プロセスの使用を必要とするかどうかを学びます。
+1. [チームの機能フラグのステータスを確認する](#dashboard--metrics)：チームが機能フラグを多く持ちすぎていないか、古い機能フラグがないかを確認します。時間の経過と共に機能フラグがどう切り替えられたかを確認します。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)